src/relentless/simulate/lammps.py

```diff
@@ -1279,7 +1350,7 @@ def process(self, sim, sim_op):
                     # consider both (i,j) and (j,i) permutations
                     if (
                         sim[self]["_rdf_params"]["exclude"]
-                        and sim[sim.initializer]["_bonds"].N != 0
+                        and sim[sim.initializer]["_bonds"] is not None
```

Comment:
This change was also necessary to address the above comment.

Reply:
Also agreed.